webapp/src/components/channel_header_button/channel_header_buttons.tsx

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, { FunctionComponent } from 'react';
+
+type HighlightProps = {
+    shouldHighlight: boolean
+};
+
+export const Highlight: FunctionComponent<HighlightProps> = ({ shouldHighlight }) => <aside>
+    <span className={shouldHighlight ? 'channel-header__icon--active' : ''}>
+        <i className='icon fa fa-list '/>
+    </span>
+</aside>
```

Comment:
I'm not sure why `aside` is being used here. You can just use parentheses here. Also a name change suggestion for component name

```suggestion
export const ChannelHeaderButton: FunctionComponent<Props> = ({ shouldHighlight }) => (
    <span className={shouldHighlight ? 'channel-header__icon--active' : ''}>
        <i className='icon fa fa-list'/>
    </span>
);
```